edb/server/pgcon/pgcon.pyx

```diff
@@ -3015,16 +3017,166 @@ cdef bytes FLUSH_MESSAGE = bytes(WriteBuffer.new_message(b'H').end_message())
 
 cdef EdegDBCodecContext DEFAULT_CODEC_CONTEXT = EdegDBCodecContext()
 
+cdef set ENUM_SETTINGS = {
```

Comment:
A comment would be nice, including a command to generate this list if you used one, so that the future maintainer knows how to sync with new upstream enums.